docs/feature-guides/clarinet-js-sdk.md

```diff
@@ -414,7 +425,7 @@ getContractsInterfaces(): Map<string, ContractInterface>
 
 Returns the interfaces of the project contracts.
```

Comment:
Some parts of this language can be revisited (in a different PR, perhaps) for tonality. cc: @max-crawford

Reply:
@sabbyanandan 
Resolving, we need to merge this PR to fix docs build